tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4190,4 +4193,135 @@ public function test_add_currency_conversion_notice_reads_from_order_meta_withou
+
+		WC_Stripe_Database_Cache::delete( 'checkout_session_' . $checkout_session_id );
+
+		$expected_amount = WC_Stripe_Helper::get_woocommerce_amount_from_stripe_amount( 1500, 'eur' );
+		$expected_rate   = wc_format_decimal( 1500 / 2000, wc_get_price_decimals() );
+
+		$this->assertStringContainsString( '<div', $output );
+		$this->assertStringContainsString( 'Currency Conversion', $output );
+		$this->assertStringContainsString( $expected_amount . ' EUR', $output );
+		$this->assertStringContainsString( $expected_rate . ' EUR', $output );
+		$this->assertStringContainsString( '</div>', $output );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_email_currency_conversion_notice_outputs_notice_with_converted_amount_and_rate is not named in camelCase. (undefined)

(CamelCaseMethodName)